nb5-proof/pom.xml

```diff
@@ -0,0 +1,134 @@
+
+  <profiles>
+      <profile>
+          <id>enable-container-tests</id>
+          <activation>
+              <activeByDefault>false</activeByDefault>
+          </activation>
+          <build>
+              <plugins>
+                  <plugin>
+                      <groupId>org.apache.maven.plugins</groupId>
+                      <artifactId>maven-failsafe-plugin</artifactId>
+                      <executions>
+                          <execution>
+                              <id>run-container-tests</id>
+                              <phase>integration-test</phase>
+                              <goals>
+                                  <goal>integration-test</goal>
+                                  <goal>verify</goal>
+                              </goals>
+                          </execution>
+                      </executions>
+                      <configuration>
+                          <forkCount>1</forkCount>
+                          <reuseForks>false</reuseForks>
+                          <includes>
+                              <include>**/*Container*Verification*.java</include>
+                              <include>**/*ContainerVerifications.java</include>
+                          </includes>
+                          <properties>
+                          </properties>
+                      </configuration>
+                  </plugin>
+              </plugins>
+          </build>
+
+      </profile>
+  </profiles>
+  <parent>
+    <artifactId>mvn-defaults</artifactId>
+    <groupId>io.nosqlbench</groupId>
+    <version>${revision}</version>
+    <relativePath>../mvn-defaults</relativePath>
+  </parent>
+
+  <artifactId>nb5-proof</artifactId>
+  <packaging>jar</packaging>
+  <name>${project.artifactId}</name>
+  <description>An end-to-end testing module for bundled drivers.</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <javadoc.name>nb5-end-to-end</javadoc.name>
+  </properties>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>io.nosqlbench</groupId>
+      <artifactId>nb5</artifactId>
+      <version>${revision}</version>
+    </dependency>
+
+      <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-annotations</artifactId>
+          <version>2.14.1</version>
+      </dependency>
+      <dependency>
+          <groupId>org.testcontainers</groupId>
+          <artifactId>testcontainers</artifactId>
+          <version>1.17.6</version>
+      </dependency>
+
+      <dependency>
+          <groupId>org.testcontainers</groupId>
+          <artifactId>cassandra</artifactId>
+          <version>1.17.6</version>
+      </dependency>
+
+  </dependencies>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+
+    <plugins>
+<!--      <plugin>-->
```

Comment:
It's present in a different test module, I thought maybe whoever put it there wanted to keep it just in case and might want to have it in another test module? but you're right if we don't need it specifically it's just dead code we're copying around. I'll remove it